samples/subsys/pm/latency/src/test.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "test.h"
+
+#include <zephyr/logging/log.h>
+#include <zephyr/pm/policy.h>
+
+LOG_MODULE_REGISTER(dev_test, CONFIG_APP_LOG_LEVEL);
+
+struct dev_test_data {
+	struct pm_policy_latency_subscription subs;
+	struct pm_policy_latency_request req;
+};
+
+static void on_latency_changed(int32_t latency)
+{
+	if (latency == SYS_FOREVER_US) {
+		LOG_INF("Latency constraint changed: none");
+	} else {
+		LOG_INF("Latency constraint changed: %"PRId32, latency);
```

Comment:
It would be nice to print it in time unit as it is done in other places.

Reply:
adjusted to print in msec (same as in message printed when setting latency constraint)